pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -112297,7 +112297,7 @@ self: {
        license = lib.licenses.isc;
        hydraPlatforms = lib.platforms.none;
        mainProgram = "futhark";
-       broken = true;
+       maintainers = [ lib.maintainers.athas ];
```

Comment:
This file is generated automatically.  Could you instead remove the package from `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml`, add yourself as a maintainer in `pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml`, and regenerated the package set using `maintainers/scripts/haskell/regenerate-hackage-packages.sh --fast`?

Reply:
Done.